packages/product-feed-luigis-box/src/Resources/views/feed.xml.twig

```diff
@@ -0,0 +1,52 @@
+{% block begin %}
+<?xml version="1.0" encoding="utf-8"?>
+<rss version="2.0" xmlns:g="http://base.google.com/ns/2.0">
+    <channel>
+{% endblock %}
+
+{% block item %}
+    <product>
+        <identity>{{ item.identity }}</identity>
+        <title>{{ item.title }}></title>
+        <url>{{ item.url }}</url>
+        <availability>{{ item.availability }}</availability>
+        <availability_rank>{{ item.availabilityRank }}</availability_rank>
+        <availability_rank_text>{{ item.availabilityRankText }}</availability_rank_text>
+        {% for categoryId, categoryNames in item.categoryNamesIndexedByCategoryId %}
+            <category{% if categoryId is same as(item.mainCategoryId) %} primary="true"{% endif %}>{{ categoryNames }}</category>
+        {% endfor %}
+        {%  if item.brand is not null -%}
+            <brand>{{ item.brand }}</brand>
+        {% endif -%}
+        <price>{{ item.price.priceWithVat|priceWithCurrencyByCurrencyId(item.currency.id) }}</price>
+        {% if item.imageLinkS is not null -%}
+            <image_link_s>{{ item.imageLinkS }}</image_link_s>
+            <image_link_m>{{ item.imageLinkM }}</image_link_m>
+            <image_link_l>{{ item.imageLinkL }}</image_link_l>
+        {% endif -%}
+        <description>{{ item.description }}</description>
```

Comment:
the description is nullable, should not this be wrapped in the `if` condition then?

Reply:
Added